session/interface.go

```diff
@@ -39,7 +47,7 @@ type Session struct {
 	ServerAddr      string
 	DevServer       bool
 	MacaroonRootKey uint64
-	Macaroon        *macaroon.Macaroon
```

Comment:
Do we need to do a "migratIon" to properly make this change? Or seems TLV saves the day here...